pkg/connectorrunner/runner.go

```diff
@@ -45,20 +45,35 @@ var ErrSigTerm = errors.New("context cancelled by process shutdown")
 
 // Run starts a connector and creates a new C1Z file.
 func (c *connectorRunner) Run(ctx context.Context) error {
+	l := ctxzap.Extract(ctx)
 	ctx, cancel := context.WithCancelCause(ctx)
 	defer cancel(ErrSigTerm)
 
 	if c.tasks.ShouldDebug() && c.debugFile == nil {
 		var err error
-		c.debugFile, err = os.Create(filepath.Join(c.tasks.GetTempDir(), "debug.log"))
+		tempDir := c.tasks.GetTempDir()
+		if tempDir == "" {
+			wd, err := os.Getwd()
+			if err != nil {
+				l.Info("unable to get the current working directory")
```

Comment:
we can't imo just info on an err that error could be anything, we usually know which error we want to ignore